chain/types/receipt.go

```diff
@@ -10,6 +10,11 @@ type MessageReceipt struct {
 	ExitCode    exitcode.ExitCode
 	ReturnValue []byte
 
-	// TODO make this an int64
-	GasUsed big.Int
+	GasUsed GasUnits
+}
+
+type GasUnits int64
+
+func (gu GasUnits) AsBigInt() big.Int {
+	return big.NewInt(int64(gu))
```

Comment:
`NewIntUnsigned(gu)`

Reply:
GasUnits are signed, right?

Reply:
Oh, yes sorry. I'm not clear why you need the conversion in this case then.

Reply:
because of `type GasUnits int64`